sites/public/cypress/integration/pages/application/preferences/all.spec.ts

```diff
@@ -0,0 +1,196 @@
+describe("applications/preferences/all", function () {
+  const route = "/applications/preferences/all"
+
+  const liveWorkLiveId = "application.preferences.options.liveWork.live.claimed"
```

Comment:
Nitpick: I think `key` suffix would be better for what this actually represents

Reply:
hmm it's HTML ID so I think id works better :)